README.md

```diff
@@ -1,11 +1,130 @@
-# amrita_pyq
+![Static Badge](https://img.shields.io/badge/ampyq-darkred?logoColor=white)
```

Comment:
Any reason for `ampyq`?

Reply:
![image](https://github.com/user-attachments/assets/41d740f5-ba95-44b5-b65f-9b31829bd6d8)
"name is in the root.go file, and it sounds good as well, so why not keep it as the app name?" was my thought process

Reply:
awsm